rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -327,13 +296,8 @@ fn test_upgrade_restricted() {
 
     // upgrade
     let upgrade_args = UpgradeArgs {
-        retrieve_btc_min_amount: Some(2000),
-        min_confirmations: None,
-        max_time_in_queue_nanos: Some(100),
         mode: Some(Mode::RestrictedTo(vec![authorized_principal])),
-        kyt_fee: None,
-        new_kyt_principal: Some(CanisterId::from(0)),
```

Comment:
kyt_principal is effectively being removed, and no longer available in the internal state. The next PR after this will rename `new_kyt_principal` to `kyt_principal`.